src/vcpkg/install.cpp

```diff
@@ -797,7 +798,8 @@ namespace vcpkg::Install
         const KeepGoing keep_going =
             to_keep_going(Util::Sets::contains(options.switches, OPTION_KEEP_GOING) || only_downloads);
         const bool prohibit_backcompat_features =
-            Util::Sets::contains(options.switches, (OPTION_PROHIBIT_BACKCOMPAT_FEATURES));
+            Util::Sets::contains(options.switches, (OPTION_PROHIBIT_BACKCOMPAT_FEATURES)) ||
```

Comment:
I think this can never be true when the option is missing in the `INSTALL_SWITCHES` list, but I have not checked that.

Reply:
Good point!